### DISCUSSION TITLE
Does `dance.run` work with counts?

### DISCUSSION THREAD

**Discussion by yerlaser** [71/dance#272] [Q&A]
I am trying to create a binding to select full lines down.
```
    {
        "key": "shift+j",
        "command": "dance.run",
        "args": {
          "commands": [
            "dance.selections.expandToLines",
            "dance.select.line.below.extend",
          ],
        },
        "when": "editorTextFocus && dance.mode == 'normal'"
    },
```

This works for simple **J**, but doesn't work with counts, e.g. **7J**
But the documentation for the `dance.run` command says it's supposed to work.

**Comment by 71** [comment]
You can "passthrough" arguments such as `count` using `$include`. [Example](https://github.com/71/dance/blob/cd05df62c2203b9fa1bee1386ed2da825ec897d4/src/commands/load-all.ts#L759).

**Comment by yerlaser** [comment]
Hmm, tried a few variations of this perhaps too naïve approach, didn't work :(

```
    {
        "key": "shift+j",
        "command": "dance.run",
        "args": {
          "commands": [
            ["dance.select.down.extend", {$include: ["count"]}],
            "dance.selections.expandToLines",
          ],
        },
        "when": "editorTextFocus && dance.mode == 'normal'"
    },
```

**Reply by 71** [reply]
Ah, sorry, indeed this worked internally but the exposed command `dance.run` did not properly support that. ad61336bff165d29c5e674f3fd3916f7d6ba8bd7 fixes it; thanks for helping me find this and sorry for the issue!

**Reply by yerlaser** [reply]
Great. Thank you very much!
